psl/psl-core/src/datamodel_connector.rs

```diff
@@ -101,10 +108,7 @@ pub trait Connector: Send + Sync {
     }
 
     fn supports_referential_action(&self, relation_mode: &RelationMode, action: ReferentialAction) -> bool {
-        match relation_mode {
-            RelationMode::ForeignKeys => self.referential_actions().contains(action),
-            RelationMode::Prisma => self.emulated_referential_actions().contains(action),
-        }
```

Comment:
We had had the right idea here already - just not applied to the auto completion :til: